components/eamxx/src/share/tests/field_utils.cpp

```diff
@@ -195,7 +200,12 @@ TEST_CASE("utils") {
     horiz_contraction<Real>(result, field00, field00);
     result.sync_to_host();
     auto v = result.get_view<Real, Host>();
-    REQUIRE(v() == (1 / sp(36) + 4 / sp(36) + 9 / sp(36)));
+    // The numerator is the sum of the squares of the first dim0 integers
```

Comment:
Maybe you could make these into vars, to be a bit more self-explanatory? Like
```c++
Real sum_n = dim0 * (dim0+1) / 2;
Real sum_n_sq = dim0 * (dim0+1) * (2*dim0+1) / 6;
```
and then use those throughout the rest of the test, since they appear a few times.

Reply:
Since you use them also in the vert_contraction, maybe turn them into small inline fcns
```c++
inline int sum_n (int n) { return n*(n+1)/2; }
inline int sum_n_sq (int n) { return n*(n+1)*(2*n+1)/6; }
```

Reply:
That's b/c you are inside a fcn already. You can't declare a function inside a function, but you can declare a lambda.